python/src/aiconfig/editor/client/src/components/ConfigNameDescription.tsx

```diff
@@ -80,11 +80,16 @@ export default memo(function ConfigNameDescription({
       if (readOnly) {
         return;
       }
-      setIsEditing(true);
-      if (event.target === nameDisplayRef.current) {
+
+      if (event.currentTarget === nameDisplayRef.current) {
```

Comment:
Why change this from `.target` to `.current`?

Reply:
So for events, the event happens on the lowest element and bubbles up to each parent element. In our case, we wrap the TextRenderer with a div that has a ref. When we click on the div, we want to toggle it to "edit" mode, to allow editing the text. Except, we don't want that to happen if the TextRenderer has rendered clickable elements (anchor or copy code button) and the click is on those elements. Otherwise, clicking the link or copy code button would immediately show the edit view instead of doing normal click behaviour